server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeProcessingSchedulerImpl.java

```diff
@@ -111,7 +111,7 @@ public void hardStopScheduling() {
     try {
       waitForInProgressWorkersToFinish(350);
     } catch (InterruptedException e) {
-      LOG.debug("Grace period of hard stop has been interrupted: {}", e);
+      LOG.debug("Grace period of hard stop has been interrupted: {}", e.getMessage());
```

Comment:
```suggestion
      LOG.debug("Grace period of hard stop has been interrupted: {}", e.getMessage(), e);
```

Reply:
I did it, 🤝.
Thank you for the feedback @pierre-guillot-gh!